yoti_python_sdk/doc_scan/session/create/filter/document_restrictions_filter.py

```diff
@@ -0,0 +1,154 @@
+from yoti_python_sdk.doc_scan.constants import (
+    DOCUMENT_RESTRICTIONS,
+    INCLUSION_BLACKLIST,
+    INCLUSION_WHITELIST,
+)
+from yoti_python_sdk.utils import YotiSerializable
+from .document_filter import DocumentFilter
+
+
+class DocumentRestriction(YotiSerializable):
+    def __init__(self, country_codes, document_types):
+        self.__country_codes = country_codes
+        self.__document_types = document_types
+
+    @property
+    def country_codes(self):
+        return self.__country_codes
+
+    @property
+    def document_types(self):
+        return self.__document_types
+
+    def to_json(self):
+        return {
+            "country_codes": self.country_codes,
+            "document_types": self.document_types,
```

Comment:
```suggestion
            "country_codes": self.__country_codes,
            "document_types": self.__document_types,
```

Could we drop the getters and do this instead? It seems like these getters are only used for unit testing purposes, but would never be called directly otherwise _(this is to reduce the code we need to test/maintain)_

Also happy if it's kept as it is, as we have done similar in other SDKs and it doesn't hurt keeping the getters

Reply:
As discussed, we can leave this as is